vocab/vocabulary.yml

```diff
@@ -0,0 +1,91 @@
+vocab:
+  - id: sl
+    value: https://www.w3.org/ns/credentials/status-list#
```

Comment:
Same comment as above: this may have to change.

Reply:
What about `https://www.w3.org/ns/credentials/status#` ?

Reply:
See my answer below

Reply:
Per [WG meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-01-10-vcwg#section3-8) `/ns/credentials/status#` should be o.k.

Reply:
Fixed in 636451dd7676b2922905dc919695054fec0af6e6